src/config.lisp

```diff
@@ -64,4 +64,6 @@ is an plist and should be written as object.")
 
 (defvar *symbol-name-function* #'symbol-name)
 
+(defvar *read-default-ht-test* 'equal
```

Comment:
Let's just call this `*read-hash-table-test*`. Please move it to to bottom of the other read dynamic variables.

Reply:
Done.